src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -17,42 +17,102 @@ export interface StatusProps {
 }
 
 export class StatusRoute extends Component<StatusProps> {
+  stateCopy = () => {
+    const { state, requestedFulfillment } = this.props.order
+    switch (state) {
+      case "SUBMITTED":
+        return "Your order has been submitted."
+      case "APPROVED":
+        return "Your order is confirmed."
+      case "FULFILLED":
+        return requestedFulfillment.__typename === "Ship"
+          ? "Your order was shipped."
+          : "Your order was picked up."
+      case "CANCELED":
+        return "Your order was canceled and refunded."
+    }
+  }
+
+  messageCopy = () => {
+    const { order } = this.props
+    const { state, requestedFulfillment } = order
+    switch (state) {
+      case "SUBMITTED":
+        return (
+          <>
+            Thank you for your order. You’ll receive a confirmation email
+            shortly. If you have questions, please contact{" "}
+            <a href="#">orders@artsy.net</a>.
```

Comment:
I believe this should be a `mailto` link

Reply:
Ah, actually based on the updated copy doc I don't think we need to show the email at all anymore 😮.

Reply:
Yep, It's removed now.